src/urlValidation/urlValidation.js

```diff
@@ -0,0 +1,37 @@
+function urlValidation(url) {
+  const cases = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
```

Comment:
this constant also can go outside the function 
motivation: you make the function shorter, i.e. easier to understand. All constants inside the function are clearly dependant on the arguments